### PR TITLE
New user info in IDP

### PR DESCRIPTION
**Pull request by sfrunza13**
to do with #307 and #290 

This changes the claims that the IDP sends us to better match what staging and prod will do and adds a user that exists in seed now that did not exist in the IDP previously. Also changes persistent name id to sAMAccountName.

### CODE REVIEW

File: config/simplesamlphp-users.php [hunk 1]
```diff
@@ -19,36 +19,34 @@
...
+            'displayname' => 'Galileo Galilei',
             'email' => 'user2@myseneca.ca',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname' => 'Galileo',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname' => 'Galilei',
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name'=> 'user2@myseneca.ca',
             'sAMAccountName'=> 'user2',
-            'http://schemas.microsoft.com/identity/claims/displayname' => 'Galileo Galilei',
+            'group'=>'mycustomdomain-faculty',
         ),
+        'user3:user3pass' => array(
```
Comment (humphd) [inline config/simplesamlphp-users.php:35]: Please add this user/password to the Markdown file where we keep track of all user accounts for testing/dev.

**Review by humphd — CHANGES_REQUESTED** [review]

**Review by humphd — APPROVED** [review]

File: config/simplesamlphp-users.php [hunk 2]
```diff
@@ -13,42 +13,32 @@
...
         'exampleauth:UserPass',
         'user1:user1pass' => array(
-            'uid' => array('1'),
-            'eduPersonAffiliation' => array('group1'),
             /*
               NOTE: we add a bunch of claims that we expect to get back from Seneca,
               and need to simulate here.
             */
-            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress' => 'user1@myseneca.ca',
+            'displayname' => 'Johannes Kepler',
```
Comment (Ririio) [inline config/simplesamlphp-users.php:20]: Is the display name the value we'll be displaying on the header?

Comment (sfrunza13) [reply config/simplesamlphp-users.php:20]: Currently we are showing sAMAccountName but we can and maybe should show display name instead, I don't know really.

Comment (Ririio) [reply config/simplesamlphp-users.php:20]: Also one more thing, what were the purpose of the schema links you removed?

**Review by Ririio — APPROVED** [review]

**Pull request merged**